pkgs/development/tools/metals/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchurl, coursier, jdk, jre, makeWrapper }:
+
+let
+  baseName = "metals";
+  version = "0.3.1";
+  deps = stdenv.mkDerivation {
+    name = "${baseName}-deps-${version}";
+    buildCommand = ''
+      export COURSIER_CACHE=$(pwd)
+      ${coursier}/bin/coursier fetch org.scalameta:metals_2.12:${version} \
```

Comment:
Holy moly, is this really stable hash-wise? If so, that must be the cleanest solution to getting deps from maven I've ever seen.